meinberlin/apps/contrib/templates/meinberlin_contrib/component_library.html

```diff
@@ -304,6 +328,282 @@ <h2 id="cl-commenting">Commenting</h2>
+                                                            </div>
+                                                            <ul class="nav navbar-nav"><li class="dropdown"><button type="button" class="dropdown-toggle" aria-haspopup="true" aria-expanded="false" data-toggle="dropdown"><i class="fa fa-ellipsis-h" aria-hidden="true"></i></button>
+                                                            </ul>
+                                                        </nav>
+                                                    </div>
+                                                </div>
+                                            </div>
+                                            <form class="general-form">
+                                                <div class="form-group">
+                                                    <textarea class="form-control" placeholder="Your comment here"></textarea>
+                                                </div>
+                                                <input value="post" class="submit-button" type="submit">
+                                            </form>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <h2 id="cl-complete">Complete</h2>
+
+                <div class="cl-example">
+                    <div class="complete">
+                        <span class="complete__warning">
+                            Complete Warning - shown on the address search for the point choser map thingy.
+                        </span>
+                    </div>
+                    <div class="complete">
+                        <ul class="complete__list">
+                            The search for an address produces the hits as list:
+                            <li>
+                                <button type="button" class="complete__item">
+                                    Petersburger Str. , 10247 Friedrichshain
+                                </button>
+                            </li>
+                            <li>
+                                <button type="button" class="complete__item">
+                                    Petersburger Str. , 10249 Friedrichshain
+                                </button>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+
+                <h2 id="cl-component-library">Component library</h2>
+
+                <div class="cl-example">
+                    <div class="cl-example">
+                        Every component in the component library has a border and some padding.
+                    </div>
+                    <div class="cl-example cl-example--grey">
+                        And some have the same background as the module pages.
```

Comment:
There seems to be some typo here.

Reply:
Do you mean grey? Gray, grey, all gray (or grey) to me!

Reply:
Nevermind. I misread.

Reply:
i think i would prefer to actually use the ".module-content" class as a wrapper if we wan't to show components in the modules context for some reason. thus if the module-content changes we will have a real representation even if the background won't be gr[ea]y anmyore
altough this is also kind of weird, as the '.module-content' does also add some layout (paddings :()

Reply:
Yes, that was my first shot, but looked sooo strange because of the padding. But I think you're right. I will change that.